POEApi.Model/ProxyMapper.cs

```diff
@@ -313,6 +325,21 @@ public static BreachType GetBreachType(JSONProxy.Item item)
             }
         }
 
+        public static LegionType GetLegionType(JSONProxy.Item item)
+        {
+            try
+            {
+                return _legionMap.First(m => item.TypeLine.Contains(m.Key)).Value;
+            }
+            catch (Exception ex)
+            {
+                Logger.Log(ex);
+                Logger.Log("ProxyMapper.GetBreachType Failed! ItemType = " + item.TypeLine);
```

Comment:
s/GetBreachType/GetLegionType

Reply:
Fixed, we should really leverage [CallerMemberName] in the logger.